### ISSUE TITLE
An existing connection was forcibly closed by the remote host when making any API calls.

### ISSUE THREAD

**Issue by jimboyuk** [stripe/stripe-dotnet#2050]
Hi

So I had recently set up a .net 4.5 solution to use version 36.6.0 of the Stripe.net library. All was working fine until I upgraded my .net solution to .net 4.6.2.  After making that change I started to receive the following error on all api calls.

An existing connection was forcibly closed by the remote host 

It seems that even though .net 4.6.2 uses TLS 1.2 as the transport protocol by default, to  get the stripe API calls to work I have to use the following line in any method that make a call to the stripe remote server.

```c#
ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

I add that line right before any api calls......for example:

```c#
var servicePlans = new PlanService();

ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;

var result = servicePlans.List();
```

Without setting the `SecurityProtocolType` to `Tls12` the call fails.

**Comment by ob-stripe** [comment]
Hi @jimboyuk, thanks for the report. Even though it shouldn't be necessary, I've prepared a patch to enable TLS 1.2 on all runtimes which should fix this issue.

**Comment by remi-stripe** [comment]
@jimboyuk We just released 34.2.0 that forces TLS 1.2 even on newer versions of the .NET framework. Can you try that version and confirm that it fixed the issue for you?
